pages/docs/reference/classes.md

```diff
@@ -295,72 +294,77 @@ class MyView : View {
+The *override*{: .keyword } modifier is required for `Circle.draw()`. If it were missing, the compiler would complain.
+If there is no *open*{: .keyword } modifier on a function, like `Shape.fill()`, declaring a method with the same signature in a subclass is illegal,
 either with *override*{: .keyword } or without it. The *open*{: .keyword } modifier has no effect when added on members of a final class (i.e.. a class with no *open*{: .keyword } modifier).
 
 A member marked *override*{: .keyword } is itself open, i.e. it may be overridden in subclasses. If you want to prohibit re-overriding, use *final*{: .keyword }:
 
 <div class="sample" markdown="1" theme="idea" data-highlight-only>
 
 ```kotlin
-open class AnotherDerived() : Base() {
-    final override fun v() { ... }
+open class Rectangle() : Shape() {
+    final override fun draw() { /*...*/ }
 }
 ```
 
 </div>
 
 ### Overriding Properties 
 
-Overriding properties works in a similar way to overriding methods; properties declared on a superclass that are then redeclared on a derived class must be prefaced with *override*{: .keyword }, and they must have a compatible type. Each declared property can be overridden by a property with an initializer or by a property with a getter method.
+Overriding properties works in a similar way to overriding methods; properties declared on a superclass 
+that are then redeclared on a derived class must be prefaced with *override*{: .keyword }, and they must have a compatible type.
+Each declared property can be overridden by a property with an initializer or by a property with a getter method.
 
 <div class="sample" markdown="1" theme="idea" data-highlight-only>
 
 ```kotlin
-open class Foo {
-    open val x: Int get() { ... }
+open class Computer {
+    open val ram: Int = 16
 }
 
-class Bar1 : Foo() {
-    override val x: Int = ...
+class GamingDesktop : Computer() {
+    override val ram: Int = 32
 }
 ```
 
 </div>
 
-You can also override a `val` property with a `var` property, but not vice versa. This is allowed because a `val` property essentially declares a getter method, and overriding it as a `var` additionally declares a setter method in the derived class.
+You can also override a `val` property with a `var` property, but not vice versa.
+This is allowed because a `val` property essentially declares a getter method,
+and overriding it as a `var` additionally declares a setter method in the derived class.
 
 Note that you can use the *override*{: .keyword } keyword as part of the property declaration in a primary constructor.
 
 <div class="sample" markdown="1" theme="idea" data-highlight-only>
 
 ```kotlin
-interface Foo {
-    val count: Int
+interface Department {
+    val employeesCount: Int
 }
 
-class Bar1(override val count: Int) : Foo
+class ITService(override val count: Int) : Department //constant number of employees
```

Comment:
Minor: i'm not sure if it is in the Kotlin code-style, but we usually insert space between `//` and comment (so, it would be `// constant number of employees`)